esp-radio/src/lib.rs

```diff
@@ -203,7 +202,33 @@ const _: () = {
     };
 };
 
-pub(crate) const CONFIG: config::Config = config::Config {
+#[derive(Debug)]
+#[cfg_attr(feature = "defmt", derive(defmt::Format))]
+/// Tunable parameters for the Wi-Fi driver
+#[allow(unused)] // currently there are no ble tunables
+pub(crate) struct Config {
```

Comment:
I wonder if we need this config struct at all?

Is there any merit in just using `esp_config_int!(usize, "ESP_RADIO_CONFIG_RX_QUEUE_SIZE")` etc directly where it's needed?

I suppose the one thing we do is print this config on init, but I'm not sure that's all that helpful 🤔